scripts/cnode-helper-scripts/prereqs.sh

```diff
@@ -170,6 +170,16 @@ if [[ "${INTERACTIVE}" = 'Y' ]]; then
   fi
 fi
 
+# Description : Add epel repository when needed
+#             : $1 = DISTRO
+#             : $2 = Epel repository VERSION_ID
+#             : $3 = pkg_opts for repo install
+add_epel_repository() {
+  if [[ "${1}" =~ Fedora ]]; then return; fi
+  echo "  Enabling epel repository..."
+  ! grep -q ^epel <<< "$(yum repolist)" && $sudo yum ${2} install https://dl.fedoraproject.org/pub/epel/epel-release-latest-"${3}".noarch.rpm > /dev/null
```

Comment:
$2 and $3 needs swapping here in yum command?

Reply:
my bad